pyproject.toml

```diff
@@ -41,4 +58,4 @@ extend-exclude = '''
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+build-backend = "poetry.core.masonry.api"
```

Comment:
New spaces at the end of lines, removed newline at the end of file. This is probably result of not applied linting.